src/main/java/nel/Client.java

```diff
@@ -27,6 +33,25 @@ public Client(Origin origin) {
     this.groups = new HashMap<String, EndpointGroup>();
   }
 
+  /**
+   * Parses a client from the contents of a <code>Report-To</code> header.
+   */
+  public static Client parseFromReportToHeader(List<String> headers, Origin origin, Instant now)
```

Comment:
Add a docstring explaining the arguments. In particular, my first reading was that "headers" is all HTTP headers, but it doesn't look like that's the case on closer inspection.

Reply:
Done